data/core/commandview.lua

```diff
@@ -274,18 +275,32 @@ function CommandView:get_suggestion_line_height()
 end
 
 
-function CommandView:update_suggestions()
-  local t = self.state.suggest(self:get_text()) or {}
+function CommandView:update_suggestions(preserve)
```

Comment:
Is there any case where `preserve` should be false?

Reply:
Yes, when typing, you don't want to preserve the selection.

Reply:
Would that not matter, as `last_change` would be `"text"`?

Reply:
Good point. Okay, will remove, and just use last change to determine what to do.